legacy/reqcounter/reqcounter_test.go

```diff
@@ -125,6 +125,31 @@ func TestFlush(t *testing.T) {
 	// Ensure the request counter did not reset.
 	require.Equal(t, uint64(66), requestCounter.Requests, "Calling Flush() reset the requests of a non-resettable request counter.")
 	require.True(t, defaultTime.Equal(requestCounter.Since), "Calling Flush() reset the timestamp of a non-resettable request counter.")
+
+	// Create a request counter that exceeded a threshold.
+	requestCounter = NewRequestCounter(600)
+	requestCounter.Threshold = 599
+	// Reset debug counter.
+	debugCalls = 0
+	// Mock logrus.Warn
+	warnCalls := 0
+	rc.Warn = func(args ...interface{}) {
+		warnCalls++
+	}
+	requestCounter.Flush()
+	// Ensure both logrus.Debug and logrus.Warn was called.
+	require.Equal(t, 1, debugCalls, "Flush() failed to trigger a debug statement after exceeding it's threshold.")
+	require.Equal(t, 1, warnCalls, "Flush() failed to trigger a warning statement after exceeding it's threshold.")
+
+	// Create a non-resetting request counter that exceeded a threshold.
+	requestCounter = NewRequestCounter(600)
+	requestCounter.Threshold = 599
+	requestCounter.Resettable = false
+	// Reset warn counter.
+	warnCalls = 0
+	requestCounter.Flush()
+	// Ensure non-resetting counters never warn on exceeding a threshold.
```

Comment:
Use the word `resettable` instead of `resetting` to make it easier to grep (use consistent language).

Reply:
Ack